src/test/callDatas.ts

```diff
@@ -15,22 +15,27 @@ export const TEST_CONTRACTS: Record<
     name: "OwnableExecutor",
     address: "0xc98B026383885F41d9a995f85FC480E9bb8bB891"
   },
-  // Smart sessions
+  // Smart sessions: TODO: update with latest address
   SmartSession: {
     chainId: 84532,
     name: "SmartSession",
-    address: "0x0ce452fD9d0a20BCC14A2C4830E22bEB7EA4C8ee"
+    address: "0x3834aD7f5f73fAd19C089a924F18e6F3417d1ac2"
   },
-  SimpleSigner: {
+  SimpleSessionValidator: {
     chainId: 84532,
-    name: "Simple Signer",
+    name: "Simple Session Validator",
     address: "0xAAAdFd794A1781e4Fd3eA64985F107a7Ac2b3872"
   },
   UniActionPolicy: {
     chainId: 84532,
     name: "UniActionPolicy",
     address: "0x28120dC008C36d95DE5fa0603526f219c1Ba80f6"
   },
+  Counter: {
```

Comment:
I thought counter was already deployed by hardhat as part of the nexus deployment. I think it is, and I think the ABI is available too.

Reply:
It's fine to keep it here. there are other mock contracts too here.